SQL/New_patches/2024-10-09-Add-French.sql

```diff
@@ -0,0 +1,4 @@
+SET NAMES utf8;
+
+INSERT INTO `language` (`language_code`, `language_label`) VALUES
+    ('fr-CA', 'Français');
```

Comment:
I thought it was agreed at the beginning of this project that we weren't going to add french as a default option in LORIS until a reasonable chunk of the translation was done, since it will look very bad if a user chooses "Français" but the majority of the interface is in English and projects in other jurisdictions might want other languages?

(I do think this insert statement should be documented somewhere for developers but I'm not entirely sure where.)